sourcebundle/builder.go

```diff
@@ -713,3 +736,20 @@ func packagePrepareWalkFn(root string, ignoreRules *ignorefiles.Ruleset) filepat
 		return nil
 	}
 }
+
+func extractVersionListFromResponse(modPackageInfos []ModulePackageInfo) versions.List {
+	vs := make(versions.List, len(modPackageInfos))
+	for index, v := range modPackageInfos {
+		vs[index] = v.Version
+	}
+	vs.Sort()
+	return vs
+}
+
+func extractVersionDeprecationsFromResponse(modPackageInfos []ModulePackageInfo) map[versions.Version]*ModulePackageVersionDeprecation {
```

Comment:
This function is very simple and only called from one site. I think it's more typical go style to inline this instead.

Reply:
Included 👍 https://github.com/hashicorp/go-slug/pull/59/commits/be5d90781af6a49d0b5f5dc5fca24e185629c14a